models/reviews.models.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db/connection');
+
+exports.selectReview = (id_param) => {
+    const review_id = parseInt(id_param.review_id.slice(1));
```

Comment:
If you don't use the `:` in the path when you make the request, you shouldn't have to do this line at all! Remove the `:` and `console.log(review_id)` here to see what comes through.

Reply:
I mean remove the `:` from the request in your tests by the way, not your app.js